SCAutolib/models/CA.py

```diff
@@ -292,6 +410,28 @@ def request_cert(self, csr: Path, username: str,
         :return: returns path to the signed certificate
         :rtype: pathlib.Path
         """
+        """
+        Creates and signs a certificate from a given CSR (Certificate Signing
+        Request) using the local CA's private key.
+        The signed certificate is stored in a predefined location (e.g.,
+        ``<root ca directory>/certs/<username>.pem``) or a specified output
+        path.
+
+        :param csr: The ``pathlib.Path`` object to the CSR file.
+        :type csr: pathlib.Path
+        :param username: The subject name to be included in the certificate.
+        :type username: str
+        :param cert_out: An optional ``pathlib.Path`` object specifying where
+                         the signed certificate should be duplicated. It can be
```

Comment:
maybe better to say "copied"? 
```suggestion
                         the signed certificate should be copied. It can be
```